config/src/main/java/org/springframework/security/config/annotation/web/configuration/WebSecurityConfiguration.java

```diff
@@ -123,16 +120,19 @@ public WebInvocationPrivilegeEvaluator privilegeEvaluator() throws Exception {
 	 *
 	 * @param objectPostProcessor the {@link ObjectPostProcessor} used to create a
 	 * {@link WebSecurity} instance
-	 * @param webSecurityConfigurers the
+	 * @param autowiredWebSecurityConfigurersIgnoreParents the
+	 * {@link AutowiredWebSecurityConfigurersIgnoreParents} containing the
 	 * {@code <SecurityConfigurer<FilterChainProxy, WebSecurityBuilder>} instances used to
 	 * create the web configuration
 	 * @throws Exception
 	 */
 	@Autowired(required = false)
 	public void setFilterChainProxySecurityConfigurer(
 			ObjectPostProcessor<Object> objectPostProcessor,
-			@Value("#{@autowiredWebSecurityConfigurersIgnoreParents.getWebSecurityConfigurers()}") List<SecurityConfigurer<Filter, WebSecurity>> webSecurityConfigurers)
+			AutowiredWebSecurityConfigurersIgnoreParents autowiredWebSecurityConfigurersIgnoreParents)
```

Comment:
Can you explain why this change is necessary? A concern I have is that setting `List<SecurityConfigurer<Filter, WebSecurity>>` is agnostic to have we got it and only the annotation instructs how to look up the value. Another concern is that `AutowiredWebSecurityConfigurersIgnoreParents` is package private.

Reply:
This change is not neccessary. I have reverted it.